### PR TITLE
SPB-620 - Updates to notebook page template to include back to home b…

### PR DESCRIPTION
**Pull request by mgough-970**
Creation of "Return to Home" link on generated notebook pages.

### CODE REVIEW

File: nbcollection/ci/merge_artifacts/template/notebook.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 				<a target="_blank" href="https://github.com/{{ page.org_name }}/{{ page.repo_name }}">
                 	<img class="logo" src="../../images/Jspectra.svg" alt="JWST Data Analysis Notebooks"/>
                 </a>
+                <a href="../../index.html" ,="" class="button" style="color: white !important">Return to Main Page</a>
```
Comment (larrybradley) [inline nbcollection/ci/merge_artifacts/template/notebook.html]: What does `,=""` do?  It doesn't look like valid HTML.

Comment (mgough-970) [reply nbcollection/ci/merge_artifacts/template/notebook.html]: Probably nothing - I copied it from an editor.  I'll remove it and test it out and see if there's a difference.

File: nbcollection/ci/merge_artifacts/template/notebook.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 				<a target="_blank" href="https://github.com/{{ page.org_name }}/{{ page.repo_name }}">
                 	<img class="logo" src="../../images/Jspectra.svg" alt="JWST Data Analysis Notebooks"/>
                 </a>
-                <a href="../../index.html" ,="" class="button" style="color: white !important">Return to Main Page</a>
+                <a href="../../index.html" class="button" style="color: white !important">Return to Main Page</a>
```
Comment (mgough-970) [inline nbcollection/ci/merge_artifacts/template/notebook.html:12]: Removed the - ,="" - string with no ill effects.

**Review by eteq — APPROVED** [review]
Wow the CI is pretty deeply broken, but I think that's not a big deal since it doesn't touch this template.  LGTM, so will go ahead and merge!

**Pull request merged**